src/main/default/lwc/descriptionStrengthIndicator/__test__/descriptionStrengthIndicator.test.js

```diff
@@ -0,0 +1,208 @@
+import { createElement } from 'lwc';
+import DescriptionStrengthIndicator from 'c/descriptionStrengthIndicator';
+
+function createTestComponent() {
+  const element = createElement('c-description-strength-indicator', {
+    is: DescriptionStrengthIndicator
+  });
+  document.body.appendChild(element);
+
```

Comment:
Just a suggestion but you're setting a lot of values on the element in the test setup but it could be a little cleaner if you pass them as arguments to the createTestComponent function as a prop argument.

```suggestion
function createTestComponent(props) {
  const element = createElement('c-description-strength-indicator', {
    is: DescriptionStrengthIndicator
  });
  document.body.appendChild(element);
  Object.keys(props).forEach((key) => element[key] = props[key]);

  return element;
}
```